src/LongTermStorage/DotnetEventProcessor/ConsumerGroupsDeployment/ServiceConfiguration.Cloud.cscfg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ServiceConfiguration serviceName="ConsumerGroupsDeployment" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceConfiguration" osFamily="4" osVersion="*" schemaVersion="2015-04.2.6">
+  <Role name="ConsumerGroups.WorkerRole">
+    <Instances count="1" />
+    <ConfigurationSettings>
+      <Setting name="Coldstorage.ConsumerGroupName" value="ColdStorage" />
+      <Setting name="Coldstorage.CheckpointStorageAccount" value="DefaultEndpointsProtocol=https;AccountName=[YourStorageAccount];AccountKey=[YourAccountKey]" />
```

Comment:
Yes you are right. I added a task in the backlog. It is probably a good idea not to merge this until we have the scripts updated.